CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ Changes since v2.35.1
 
 ## v2.35.1 "Selkämerenkatu +1" 2024-01-16
 
+**NB: This release contains migrations!**
+
+### Changes
+- The EGA implementation has been removed as unnecessary (#3239).
```

Comment:
changelog could use additional context: the implementation was related to entitlement push (v2), `docs/architecture/018-ega-integration.md`, that plugin support may be added in the future.

Reply:
I'm not sure there is need to add more detail here as it wasn't actually used by anyone.

Reply:
that may be the case, but i think it's helpful (to the reader) to give context, provided the message does not get harder to read. `"removed as unnecessary"` can be quite a few things - unnecessary for who, when did it become unnecessary, why did it become unnecessary, what does it mean it is unnecessary. 

that being said, proper communication is difficult, and there is a link to github issue where reader can find extra details